Automattic/pocket-casts-ios
# Settings Sync: Add filesSortOrder setting

| 📘 Part of: #1400 |
|:---:|

* Adds a new synced setting for `fileSortOrder`
* Adds new sort methods for ascending alphabetical, and durations

<img src="https://github.com/Automattic/pocket-casts-ios/assets/3250/f929d931-c64b-4f2c-b648-6fe1f386aec1" width=300>

## To test

Enable the `newSettingsStore` and `settingsSync` feature flags. Restart the app.

* D1: Navigate to Profile > Files
* D1: Add files by dragging and dropping on the simulator
* D1: Tap the ⋯ button in the upper right
* D1: Change the "Sort By" value
* D1: Refresh Profile
* D2: Refresh Profile
* D2: Navigate to Profile > Files
* D2: Verify that sort order is correct
* D2: Tap the ⋯ button in the upper right
* D2: Verify that sort type is correct

## Checklist
